backend/pokemons/helpers.py

```diff
@@ -18,3 +19,23 @@ def save_pokemon(poke_id):
     )
 
     return pokemon
+
+
+def get_pokemons_points_sum(poke_ids):
+    points_sum = 0
+    for poke_id in poke_ids:
+        pokemon = Pokemon.objects.filter(poke_id=poke_id).first()
+
+        if pokemon:
+            points_sum += pokemon.attack + pokemon.defense + pokemon.hit_points
+        else:
+            if not pokemon_exists(poke_id):
+                raise PokemonNotFound(poke_id)
```

Comment:
Where's the test for this? Remember to use TDD!

Reply:
Haha, thanks for the catch and for remember me about TDD!